views/js/controller/Delivery/monitoring.js

```diff
@@ -434,13 +434,33 @@ define([
                         reasonRequired: true,
                         categoriesSelector: cascadingComboBox(categories[actionName] || {})
                     });
+                    var warningAction;
+                    var warningMessage;
+                    var warningReasons;
 
                     if (!config.allowedResources.length) {
-                        if (_selection.length > 1) {
-                            feedback().warning(__('No report available for these test sessions'));
+                        if (config.deniedResources) {
+                            if (config.deniedResources.length > 1) {
+                                warningAction = __('Cannot') + ' ' + actionName + ' ' + __('these test sessions.');
+                                warningReasons += _.map(config.deniedResources, function (deniedResource, index) {
+                                    var opener = (index > 0 ? __('test') : __('Test')) + ' ';
+                                    return opener + deniedResource.id + ' ' + deniedResource.reason;
+                                }).join(',') + '.';
+                                warningMessage = warningAction + ' ' + warningReasons;
+                            } else {
+                                warningAction = __('Cannot') + ' ' + actionName;
+                                warningReasons = __('this test session') + ' ' + config.deniedResources[0].reason;
+                                warningMessage = warningAction + ' ' + __('because') + ' ' + warningReasons;
```

Comment:
Please do not cut the sentences that should be translated. Use parametric sentences instead. If needed you may also use a map of sentences, like described earlier.

When building a message that should be translated, always thing about the translation it could have.